packages/frontend/src/translations/en.global.json

```diff
@@ -1890,8 +1890,16 @@
             "desc": "You'll need this <strong>password</strong> to securely import your accounts. Copy or write it down until you've completed transferring your accounts.",
             "title": "Password required to import accounts"
         },
+        "selectSenderMigration": {
+            "title": "Select a migration type to export",
+            "toExtension": "Display with QR Code",
+            "withQrCode": "Display with QR Code"
```

Comment:
<img width="533" alt="image" src="https://user-images.githubusercontent.com/6027014/198905056-9cbd09be-19df-4644-b230-91ee44cf34bb.png">

duplicated label, need to be fixed